llvm/lib/Target/RISCV/RISCVExpandPseudoInsts.cpp

```diff
@@ -395,6 +409,87 @@ bool RISCVExpandPseudo::expandCoreVBitManip(MachineBasicBlock &MBB,
   return true;
 }
 
+bool RISCVExpandPseudo::expandCoreVClip(llvm::MachineBasicBlock &MBB,
+                                        MachineBasicBlock::iterator MBBI) {
+  DebugLoc DL = MBBI->getDebugLoc();
+  Register DstReg = MBBI->getOperand(0).getReg();
+  Register I = MBBI->getOperand(1).getReg();
+  uint64_t J = MBBI->getOperand(2).getImm();
+
+  unsigned Opcode;
+  switch (MBBI->getOpcode()) {
+    case RISCV::CV_CLIPU_PSEUDO:
+      Opcode = RISCV::CV_CLIPU;
+      break;
+    case RISCV::CV_CLIP_PSEUDO:
+      Opcode = RISCV::CV_CLIP;
+      break;
+    default:
+      llvm_unreachable("unknown instruction");
+  }
+  const MCInstrDesc &Desc = TII->get(Opcode);
+  BuildMI(MBB, MBBI, DL, Desc, DstReg).
+      addReg(I).
```

Comment:
Probably `.addReg(I)` instead of `addReg(I).`?

Reply:
Agreed